src/OrchardCore.Modules/OrchardCore.Apis.GraphQL/GraphQLMiddleware.cs

```diff
@@ -165,13 +165,10 @@ private async Task ExecuteAsync(HttpContext context, ISchemaFactory schemaServic
                 _.Inputs = request.Variables.ToInputs();
                 _.UserContext = _settings.BuildUserContext?.Invoke(context);
                 _.ExposeExceptions = _settings.ExposeExceptions;
+                _.ValidationRules = (_settings.ValidationRules ?? Enumerable.Empty<IValidationRule>()).Concat(DocumentValidator.CoreRules());
             });
 
-            var httpResult = result.Errors?.Count > 0
-                ? HttpStatusCode.BadRequest
-                : HttpStatusCode.OK;
-
-            context.Response.StatusCode = (int)httpResult;
+            context.Response.StatusCode = (int) HttpStatusCode.OK;
```

Comment:
why?

Reply:
Surely with a permissions check now as well you need to return the http status code for forbidden?

Reply:
@Jetski5822 did you read the comment about it in the PR description ?

Reply:
nope :P